src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,57 @@
+package duke;
+
+import java.io.IOException;
+import duke.parser.Parser;
+import duke.task.TaskList;
+import duke.ui.Ui;
+import duke.storage.Storage;
+
+/**
+ * Represents the main program. It contains an instance of the Storage class to load data from the user's disk
+ * and save data to the user's disk according to the changes made. It also contains an instance of the TaskList class to function
+ * as a task manager for the user. The instantiation of the Ui class is to facilitate user interaction.
+ */
+public class Duke {
+    private Storage storage;
+    private TaskList taskList;
+    private Ui ui;
+
+    /**
+     * Constructs an instance of the Duke class. The instance will contain an Ui object for interaction with the user
+     * and a Storage object to load previously saved data and save new changes made to the data on the user's local disk.
+     */
```

Comment:
[Code Quality] In this case, it's probably better to remove the comment to reduce redundancy. It should already be quite clear what the method does.